### ISSUE TITLE
eslint fails due to jsx-a11y plugin failing

### ISSUE THREAD

**Issue by rcarigna** [AtomLinter/linter-eslint#1377]
## Issue Type
Bug?

## Issue Description
Eslint no longer works in atom. I restarted my Macbook after a security update and now atom complains: 
`Failed to load plugin 'jsx-a11y' declared in '.eslintrc.js » eslint-config-airbnb » /[directory_path]/node_modules/eslint-config-airbnb/rules/react-a11y.js': Cannot read property 'href' of undefined`
(I have put substituted [directory_path] manually)
Linting errors are no longer displayed at the bottom or on save or fixed on save. I've tried uninstalling and reinstalling the package as well as restarting. Not sure what's happened. Any guidance would be appreciated!

## Bug Checklist
- [ ✔️] Restart Atom
- [ ✔️] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [ ✔️] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Atom version: 1.53.0
linter-eslint version: 8.5.5
ESLint version: 7.13.0
Hours since last Atom restart: 0.4
Platform: darwin
Using local project ESLint from: /[directory_path]/node_modules/eslint
Current file's scopes: [
  "source.js",
  "string.quoted"
]
linter-eslint configuration: {
  "autofix": {
    "fixOnSave": true,
    "ignoreFixableRulesWhileTyping": false,
    "rulesToDisableWhileFixing": []
  },
  "global": {
    "useGlobalEslint": false,
    "eslintrcPath": "",
    "globalNodePath": ""
  },
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.flow",
    "source.babel",
    "source.js-semantic"
  ],
  "lintHtmlFiles": false,
  "disabling": {
    "disableWhenNoEslintConfig": true,
    "rulesToSilenceWhileTyping": []
  },
  "advanced": {
    "disableEslintIgnore": false,
    "disableFSCache": false,
    "showRuleIdInMessage": true,
    "eslintRulesDirs": [],
    "localNodeModules": ""
  }
}
```
